desktop/aseprite.appdata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">aseprite.desktop</id>  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>Aseprite</name>
+  <summary>Animated sprite editor and pixel art tool</summary>
+  <description>
+    <p>Aseprite lets you create 2D animations for videogames. From sprites, to pixel-art, retro style graphics, and whatever you like about the 8-bit (and 16-bit) era.</p>
+  </description>
+  <url type="homepage">http://www.aseprite.org/</url>
+  <screenshots>
+    <screenshot>http://i.imgur.com/1myue2n.gif</screenshot>
```

Comment:
Something like http://davidcapello.com/images/aseprite.png without the Windows 10 border would be great.